sentry/src/main/java/io/sentry/profilemeasurements/ProfileMeasurement.java

```diff
@@ -0,0 +1,150 @@
+package io.sentry.profilemeasurements;
+
+import io.sentry.ILogger;
+import io.sentry.JsonDeserializer;
+import io.sentry.JsonObjectReader;
+import io.sentry.JsonObjectWriter;
+import io.sentry.JsonSerializable;
+import io.sentry.JsonUnknown;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Internal
+public final class ProfileMeasurement implements JsonUnknown, JsonSerializable {
+
+  public static final String ID_FROZEN_FRAME_RENDERS = "frozen_frame_renders";
+  public static final String ID_SLOW_FRAME_RENDERS = "slow_frame_renders";
+  public static final String ID_SCREEN_FRAME_RATES = "screen_frame_rates";
+  public static final String ID_UNKNOWN = "unknown";
+
+  public static final String UNIT_HZ = "hz";
+  public static final String UNIT_NANOSECONDS = "nanosecond";
```

Comment:
yep, is it fine if i add the `hz` to `MeasurementUnit.Duration`?

Reply:
For me it's fine, but could you check with someone from the ingest/your backend team, because it's derived from the relay docs https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html

Maybe we should also introduce this unit there, since it's pre-defined and expected by backend?

Reply:
Sure, let me ask

Reply:
@romtsn I can confirm that we are not using the `MetricUnit` on the profiling backend
So, should we keep the units separated?